src/App.js

```diff
@@ -14,6 +15,7 @@ class App extends Component {
     return(
       <div className="App">
         <Header />
+        <Movies moviesList={this.state.movies} />
```

Comment:
Looks great, once we get our fetch call in App, we should be able to see all of these movies.